### PR TITLE
Added support for Spintax syntax in reply

### PR DESCRIPTION
**Pull request by chprasad893**
Hi Arpit,

I have added support for Spintax in reply for both response and noMatch fields using [mel-spintax](https://www.npmjs.com/package/mel-spintax) package.
I hope it will add value to this project and meets your coding standards.

Thank You.

### CODE REVIEW

**Comment by vasani-arpit** [comment]
Thank you. Will look into it and merge.

File: src/inject.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 WAPI.waitNewMessages(false, (data) => {
-    console.log(data)
-    data.forEach((message) => {
+    console.log(data);
+    data.forEach(async (message) => {
```
Comment (pavanmehta91) [inline src/inject.js]: this won't await. We'll have to convert the loop into `for(i=0; i<data.length; i++)` style

Comment (chprasad893) [reply src/inject.js]: Thank you for pointing the mistake. I have made the changes please review.

**Comment by ryahpalma** [comment]
@chprasad893 This function is a come in handy. Do you know how to implement the spintax for send multiple images?

**Comment by chprasad893** [comment]
> @chprasad893 This function is a come in handy. Do you know how to implement the spintax for send multiple images?
> 
> I managed to implement it in an amateur way. it works, but isn't clean.
> 
> ```
> if ((exactMatch || PartialMatch).file != undefined) {
>         files = await resolveSpintax((exactMatch || PartialMatch).file)
>         window.getFile (files)
> ```



> @chprasad893 This function is a come in handy. Do you know how to implement the spintax for send multiple images?
> 
> I managed to implement it in an amateur way. it works, but isn't clean.
> 
> ```
> if ((exactMatch || PartialMatch).file != undefined) {
>         files = await resolveSpintax((exactMatch || PartialMatch).file)
>         window.getFile (files)
> ```

window.getFile will get only one file right? When you say you want to send multiple images from the above code, I am assuming you want to send different images based on the spintax right?

File: src/inject.js [hunk 2]
```diff
@@ -1,7 +1,8 @@
-WAPI.waitNewMessages(false, (data) => {
-    console.log(data)
-    data.forEach((message) => {
+WAPI.waitNewMessages(false, async (data) => {
+    console.log(data);
```
Comment (pavanmehta91) [inline src/inject.js]: remove log

Comment (chprasad893) [reply src/inject.js]: Done

**Comment by vasani-arpit** [comment]
Thank you all. Will test it out and merge it.

**Pull request merged**